pkg/api/customization/clusterregistrationtokens/formatter.go

```diff
@@ -15,7 +15,7 @@ const (
 	insecureCommandFormat = "curl --insecure -sfL %s | kubectl apply -f -"
 	nodeCommandFormat     = "sudo docker run -d --privileged --restart=unless-stopped --net=host -v /etc/kubernetes:/etc/kubernetes -v /var/run:/var/run %s --server %s --token %s%s"
 
-	windowsNodeCommandFormat = `PowerShell -NoLogo -NonInteractive -Command "& {docker run -v c:/:c:/host %s bootstrap --server %s --token %s%s | iex}"`
+	windowsNodeCommandFormat = `PowerShell -NoLogo -NonInteractive -Command "& {docker run -v c:/:c:/host %s%s bootstrap --server %s --token %s%s | iex}"`
```

Comment:
Seems weird for there to be no space between `%s%s`

Reply:
@thxCode what about this? is the lack of a space here a bug or intentional?

Reply:
The below logic is: 
```
                // for windows
		var agentImageDockerEnv string
		if settings.SystemDefaultRegistry.Get() != "" {
			// patch the AGENT_IMAGE env
			agentImageDockerEnv = fmt.Sprintf("-e AGENT_IMAGE=%s ", agentImage)
		}
		resource.Values["windowsNodeCommand"] = fmt.Sprintf(windowsNodeCommandFormat,
			agentImageDockerEnv,
			agentImage,
			rootURL,
			token,
			ca)
```
I want to make the format argument `agentImageDockerEnv` explicitly via `%s%s`.